Assets/ConveyorUpdate.cs

```diff
@@ -0,0 +1,22 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class ConveyorUpdate : MonoBehaviour {
+	
+	public bool conveyorMovedL = false;
```

Comment:
oh nvm I'm not that clear on what these bools do, maybe hasMovedUp will be a better name?